content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -476,6 +476,38 @@ supported for spring web versions 3.1+. To learn more about the OpenTelemetry
 `RestTemplate` interceptor, see
 [opentelemetry-spring-web-3.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-web/spring-web-3.1/library).
 
+The following ways of creating a `RestTemplate` bean are supported:
+
+```java
+@Bean
+public RestTemplate restTemplate() {
+    return new RestTemplate();
+}
+```
+
+```java
+public MyService(RestTemplateBuilder restTemplateBuilder) {
+    this.restTemplate = restTemplateBuilder.build();
+}
+```
+
+Also provides autoconfiguration for the `RestClient` web client.
+
+The following ways of creating a `RestClient` bean are supported:
```

Comment:
```suggestion
The following ways of creating a `RestClient` are supported:
```
I would remove _bean_ because a bean is not created in the second case.